### PR TITLE
[Event] Code in the Dark Thailand #1: BKK 2018

### PR DESCRIPTION
**Pull request by hexadi**

### CODE REVIEW

File: data/2018-10/codeinthedark.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+topics:
+  - Frontend
+  - HTML
+  - CSS
+location:
+  title: cøde CRAFT Bar & Studio
+  url: https://www.google.com/maps/place/c%C3%B8de+CRAFT+Bar+%26+Studio/@13.8065307,100.5223342,15z/data=!4m2!3m1!1s0x0:0xe9eec03b79e68253?sa=X&ved=2ahUKEwic4camq-XdAhUKvo8KHQrmB9sQ_BIwGHoECAoQCw
+links:
+  - type: website
+    url: https://www.facebook.com/StupidHackTH/
```
Comment (dtinth) [inline data/2018-10/codeinthedark.md]: This link is not needed — this event is not really affiliated with StupidHackTH, I am just reusing its Event Pop organizer account ;)

Comment (PH9) [reply data/2018-10/codeinthedark.md]: ok

Comment (dtinth) [reply data/2018-10/codeinthedark.md]: Did you forget to remove this link item from the data file?

**Comment by dtinth** [comment]
Hello, could you address the requested changes? Otherwise, I might have to close this PR as invalid. Thanks!

**Comment by dtinth** [comment]
Closing as invalid due to inactivity.

**Pull request closed without merging**